webpack.production.config.js

```diff
@@ -92,5 +93,11 @@ module.exports = {
     new ScriptExtHtmlWebpackPlugin({
       defaultAttribute: 'defer'
     }),
+    new CopyWebpackPlugin([
```

Comment:
Adding `CopyWebpackPlugin` in the `Webpack` configurations is a good idea, but please put new assets a good names folder, like `imgs` or `images` for `img`. Just like you see I put bundled `CSS` and `JS` files in separated folders inside `assets` folder.

All of your changes are cute and ready to merge. Thanks 👍

Reply:
I didn't mean images specifically for assets's use case,
it can contains any static file such as audio, video, etc
if we need any folder structure on build
we can simply just create the folders in assets folder at ***src***
and they will be copied to ***dist***

Reply:
I will put those logos inside images folder.
I think it will fix the mess at the dist folder

Reply:
Thanks for your description and comment. I wait for your new commits. 💘